cmd/minishift/cmd/start.go

```diff
@@ -416,6 +420,20 @@ func setSubscriptionManagerParameters() {
 	cluster.RegistrationParameters.Password = viper.GetString(startFlags.Password.Name)
 }
 
+func checkRegistrationCredential(host *host.Host) {
+	needRegistration, _ := registration.NeedRegistration(host)
+	if needRegistration {
+		if cluster.RegistrationParameters.Username == "" {
+			username := pkgUtil.ReadInputFromStdin("Username to register to subscription-manager")
```

Comment:
@praveenkumar Is there any way we can check which `register` we are using i.e. `Redhat` for for a RHEL ISO? If so we should make the string as `Enter Red Hat Username to register the VM`